conference.js

```diff
@@ -765,46 +765,43 @@ export default {
     muteAudio(mute, showUI = true) {
         // Not ready to modify track's state yet
         if (!this._localTracksInitialized) {
+            muteLocalAudio(mute);
             this.setAudioMuteStatus(mute);
+
             return;
-        } else if (localAudio && localAudio.isMuted() === mute) {
+        } else if (this.isLocalAudioMuted() === mute) {
             // NO-OP
             return;
         }
 
-        const maybeShowErrorDialog = (error) => {
-            if (showUI) {
-                APP.UI.showMicErrorNotification(error);
-            }
-        };
-
-        if (!localAudio && this.audioMuted && !mute) {
+        if (!this.localAudio && !mute) {
             createLocalTracks({ devices: ['audio'] }, false)
                 .then(([audioTrack]) => audioTrack)
                 .catch(error => {
-                    maybeShowErrorDialog(error);
+                    if (showUI) {
+                        APP.UI.showMicErrorNotification(error);
+                    }
 
                     // Rollback the audio muted status by using null track
                     return null;
                 })
                 .then(audioTrack => this.useAudioStream(audioTrack));
         } else {
-            const oldMutedStatus = this.audioMuted;
-
-            muteLocalAudio(mute)
-                .catch(error => {
-                    maybeShowErrorDialog(error);
-                    this.setAudioMuteStatus(oldMutedStatus);
-                    APP.UI.setAudioMuted(this.getMyUserId(), this.audioMuted);
-                });
+            muteLocalAudio(mute);
         }
     },
     /**
      * Returns whether local audio is muted or not.
      * @returns {boolean}
      */
     isLocalAudioMuted() {
-        return this.audioMuted;
+        // If the tracks are not ready, read from base/media state
+        return this._localTracksInitialized
+            ? isLocalTrackMuted(
```

Comment:
The same comment but about this.localAudio. I now understand it can be undefined, so I withdraw my observation :-)